src/builder/pod/volume.rs

```diff
@@ -343,6 +351,18 @@ impl SecretOperatorVolumeSourceBuilder {
             );
         }
 
+        if let Some(password) = &self.tls_pkcs12_password {
+            // The `tls_pkcs12_password` is only used for PKCS12 stores.
+            if Some(SecretFormat::TlsPkcs12) != self.format {
+                warn!("A TLS PKCS12 password was set but ignored because the specified format was not set to {}", SecretFormat::TlsPkcs12.to_string())
```

Comment:
We can add what the actual format is here:
```suggestion
                warn!(format.actual = ?self.format, format.expected = ?Some(SecretFormat::TlsPkcs12), "A TLS PKCS12 password was set but ignored because another format was requested")
```